db/migrate/20220916082459_rename_cinema_halls_to_halls.rb

```diff
@@ -0,0 +1,5 @@
+class RenameCinemaHallsToHalls < ActiveRecord::Migration[7.0]
```

Comment:
Since this is PR you could simply rollback these two migrations, remove the second one and just rename the first

```
rails db:rollback STEP=2
```

Reply:
That makes perfect sense, thanks. I just had this idea in my mind that I shouldn't change already existing migrations, but this is PR, you are right.